crypto/crypto.go

```diff
@@ -66,13 +66,13 @@ func trimLeftZeros(b []byte) []byte {
 }
 
 // ValidateSignatureValues checks if the signature values are correct
-func ValidateSignatureValues(v byte, r, s *big.Int) bool {
+func ValidateSignatureValues(v, r, s *big.Int) bool {
 	// TODO: ECDSA malleability
 	if r == nil || s == nil {
 		return false
 	}
 
-	if v > 1 {
+	if v.Cmp(big.NewInt(1)) == 1 || v.Cmp(big.NewInt(0)) == -1 {
```

Comment:
You can rely on already present variable `big1`. Note that you would need to introduce `big0` variable, the same way we have `big1` (and also this would clash a bit with #1195, but hopefully it isn't a major problem)
Also makes more sense to first check lower and then upper bound :)

```suggestion
	if v.Cmp(big0) == -1 || v.Cmp(big1) == 1 {
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1207/commits/18fc03393b8272b15d3b90ae0dc9eb6a8bee607c